langstream-cli/src/test/java/ai/langstream/cli/commands/applications/UIAppCmdTest.java

```diff
@@ -15,17 +15,41 @@
  */
 package ai.langstream.cli.commands.applications;
 
-import static org.junit.jupiter.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.attribute.PosixFilePermission;
+import java.util.Set;
 import org.junit.jupiter.api.Test;
 
 class UIAppCmdTest {
 
+    static void writeExecutableFile(String fname, String contents) throws IOException {
+
+        var path = Files.writeString(Path.of(fname), contents);
+
+        Files.setPosixFilePermissions(
+                path, Set.of(PosixFilePermission.OWNER_READ, PosixFilePermission.OWNER_EXECUTE));
+    }
+
     @Test
-    void openBrowser() {
+    void openBrowser() throws IOException {
+        // write test file
+        String fname = "/tmp/testfile";
```

Comment:
This test cannot pass on a Window machine.

I think that we can remove it.
Also, it is creating a fixed path, so you cannot run the same test in parallel (not a big deal, but it worth mentioning it)